packages/airnode-node/src/evm/handlers/initialize-provider.ts

```diff
@@ -109,6 +144,7 @@ export async function initializeProvider(
   const authAndTxCountPromises: readonly ParallelPromise[] = [
     fetchAuthorizations(state5),
     fetchTransactionCounts(state5),
+    fetchCrossChainAuthorizations(state5),
```

Comment:
If `fetchCrossChainAuthorizations` is plural and fetches from multiple chains, I think `fetchAuthorizations` should be renamed to singular.

Reply:
They're both plural as for each, authorizations are being fetched for an array of requests i.e. `Request<ApiCall>[]`

Reply:
But `fetchAuthorizations` returns `data: AuthorizationByRequestId` and the `fetchCrossChainAuthorizations` returns `data: AuthorizationByRequestId[]`

Reply:
Yeah, that sounds a bit better to me. (I see your point though).

Reply:
👍 Thanks for bringing it up and for the discussion, I think this will add clarity.